packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -73,6 +75,28 @@ interface Label {
   name: string;
 }
 
+/**
+ * Reduces the list of PRs to search by half if there are more than 100 entries at a time
+ * @returns an array of Datastore PRs
+ */
+handler.maybeReducePRList = function maybeReducePRList(prs: DatastorePR[]) {
+  if (prs.length >= MAX_ENTRIES) {
+    if (Date.now() % 2 === 0) {
```

Comment:
Rather than hard-coding `2`, which I think will quickly bump into scaling issues again as we continue to scale, we'd do better to instead pick a work size, like `25`, and then use this as our modulus:

```
const modulus = parseInt(prs.length / MAX_ENTRIES)
prs = prs.filter(x => new Date(x.created!).getTime() % modulus === 0);
logger.info(
    `Too many entries in Datastore table; examining first ${prs.length}`
);
```